test/db/cmd/cmd_w

```diff
@@ -233,6 +233,51 @@ nop
 EOF
 RUN
 
+NAME=wai
+FILE==
+ARGS=-a x86 -b 64
+CMDS=<<EOF
+wx 488d850678563412
+s 0
+pi 1
+p8 7
+s 0
+wai "mov rax, rbx"
+s 0
+pi 1
+p8 7
+EOF
+EXPECT=<<EOF
+lea rax, qword [rbp+0x34567806]
+488d8506785634
+mov rax, rbx
+4889d890909090
```

Comment:
this should output `mov;nop;nop;..`
instead of using `pi` we should use `pD <n-bytes>`